scripts/install.sh

```diff
@@ -27,9 +27,9 @@ function is_python_version_ok() {
 	if which python3 2>&1 >/dev/null;
 	then
 		version=`python3 -V 2>/dev/null`
-		if [[ -z `echo $version|grep -E 'Python 3\.[(6789]\.[0-9]+'` ]];
+		if [[ -z `echo $version|grep -E 'Python 3\.[(78910]\.[0-9]+'` ]];
```

Comment:
@marcofavorito I suspect that's not quite right :D

Reply:
nope :) That will match `3.7`, `3.8`, `3.9`, `3.1`, and `3.0`.

I suggest: 
```
Python 3\.(7|8|9|10)\.[0-9]+
```

Test: https://regex101.com/r/FWRBtV/1

Reply:
(btw, the previous regex included a spurious `(` character, which would allow the string e.g. `Python 3.(.0`). Test: https://regex101.com/r/zw0Dn3/1